src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -598,21 +603,30 @@ describe('Step Packages', () => {
         await searchForAvailablePackages(searchboxAvailable, 'test');
         expect(getPackages).toHaveBeenCalledTimes(1);
 
-        const availablePackages = screen.getByTestId('available-pkgs-list');
-        expect(availablePackages.children.length).toEqual(3);
+        let availablePackagesItems = within(availablePackagesList).getAllByRole('option');
+        expect(availablePackagesItems).toHaveLength(3);
 
         screen.getByRole('option', { name: /testPkg test package summary/ }).click();
         screen.getByRole('button', { name: /Add selected/ }).click();
 
-        const chosenPackages = screen.getByTestId('chosen-pkgs-list');
-        expect(availablePackages.children.length).toEqual(2);
-        expect(chosenPackages.children.length).toEqual(1);
+        availablePackagesItems = within(availablePackagesList).getAllByRole('option');
+        expect(availablePackagesItems).toHaveLength(2);
+
+        let chosenPackagesItems = within(chosenPackagesList).getAllByRole('option');
+        // Knowing if it is in document isn't enough. We want a specific length of 1 so ignore rule.
+        // eslint-disable-next-line jest-dom/prefer-in-document
+        expect(chosenPackagesItems).toHaveLength(1);
 
         searchboxAvailable.click();
         await searchForAvailablePackages(searchboxAvailable, 'test');
         expect(getPackages).toHaveBeenCalledTimes(2);
-        expect(availablePackages.children.length).toEqual(2);
-        expect(chosenPackages.children.length).toEqual(1);
+
+        availablePackagesItems = within(availablePackagesList).getAllByRole('option');
+        chosenPackagesItems = within(chosenPackagesList).getAllByRole('option');
+        expect(availablePackagesItems).toHaveLength(2);
+        // Knowing if it is in document isn't enough. We want a specific length of 1 so ignore rule.
+        // eslint-disable-next-line jest-dom/prefer-in-document
+        expect(chosenPackagesItems).toHaveLength(1);
```

Comment:
I guess better still would be to look for the package name in the chosen/available package list. This preserves current coverage so it's ok though.

Reply:
Good point. I now also check for the package name within the chosen packages list.